### ISSUE TITLE
Game crashes when taking mega screenshot

### ISSUE THREAD

**Issue by Konerak** [anegostudios/VintageStory-Issues#844]
**Platform:** Windows 
**Modded:** No 

### Description
Took a screenshot while sneaking (SHIFT F12), game crashed.

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.14.8 (Stable)31/03/2021 12:19:50: Critical error occurred
System.Exception: FBO FindBright: One or more attachment points are not framebuffer attachment complete. This could mean there’s no texture attached or the format isn’t renderable. For color textures this means the base format must be RGB or RGBA and for depth textures it must be a DEPTH_COMPONENT format. Other causes of this error are that the width or height is zero or the z-offset is out of range in case of render to volume.
   bij Vintagestory.Client.NoObf.ClientPlatformWindows.CheckFboStatus(FramebufferTarget target, String fbtype)
   bij Vintagestory.Client.NoObf.ClientPlatformWindows.SetupDefaultFrameBuffers()
   bij Vintagestory.Client.NoObf.ClientPlatformWindows.RebuildFrameBuffers()
   bij Vintagestory.Client.NoObf.SystemScreenshot.takeMegaScreenshot(KeyCombination t1)
   bij _BF9C5peJeQnuKaRgXqOqWSuhdFD._x3OabvNuiGbq0NHfZtrnj9x3CqC(KeyEvent , IWorldAccessor , IPlayer , Boolean , Boolean , Boolean )
   bij _BF9C5peJeQnuKaRgXqOqWSuhdFD._aoWYL30EA75zbwIchaMIsytuXXw(KeyEvent , IWorldAccessor , IPlayer )
   bij Vintagestory.Client.NoObf.ClientMain.OnKeyDown(KeyEvent args)
   bij Vintagestory.Client.NoObf.ClientPlatformWindows.game_KeyDown(Object sender, KeyboardKeyEventArgs e)
   bij System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bij System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bij OpenTK.Platform.NativeWindowBase.OnKeyDown(Key key, Boolean repeat)
   bij OpenTK.Platform.Windows.WinGLNative.HandleKeyboard(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   bij OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   bij OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   bij OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   bij OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bij _YrS2W1MoaeqC4xqH8ItdZB6vADN._8t1BmEWVlCT9mjx5M0vFIzU9tif(_7T00Rjby4si0wyEBez5dXNqLm7h , String[] )
   bij _ejKuc6cg0k8V39o8jDIzquyczbO._8t1BmEWVlCT9mjx5M0vFIzU9tif(ThreadStart )
-------------------------------
```

**Comment by copygirl** [comment]
Pressing Shift-F12 takes a "mega screenshot" with a very high resolution. It might be that your system isn't able to do this. Either way, the game should handle this and not crash. Possibly related crash: #114

**Comment by wdsaeghe** [comment]
Possible, but this is an i7-8700K 16GB RAM GTX 1080 Ti system, and the game is installed on my SSD.

Are you sure the related number is correct? It says "If your inventory is extended past what the GUI allows (With mods), you cannot scroll down through it." - looks unrelated.

**Comment by copygirl** [comment]
Ah sorry, I must have copied the wrong issue number. Fixed it now.

It's just my guess that it could have to do with not enough graphics memory being available (on top of what is used for the game normally) to allocate the needed framebuffers to render a mega screenshot.

**Comment by Konerak** [comment]
Maybe, though imo less likely since the system is quite beefy. 

Maybe not all objects were loaded yet (draw distance, just moved up a hill, liked the view, took a screenshot) and it tried taking a screenshot with objects still loading? 

Not urgent anyway :)

**Comment by tyronx** [comment]
I think this is now fixed in v1.15, otherwise please reopen
